wsproto/frame_protocol.py

```diff
@@ -59,11 +59,23 @@ class Opcode(IntEnum):
     """
     RFC 6455, Section 5.2 - Base Framing Protocol
     """
+
+    #: Contiuation frame
     CONTINUATION = 0x0
+
+    #: Text message
     TEXT = 0x1
+
+    #: Binary message
     BINARY = 0x2
+
+    #: Close frame
     CLOSE = 0x8
+
+    #: Ping frame
     PING = 0x9
+
+    #: Pong frame
```

Comment:
@Kriechi I'm not sure these comments add value, in the context it is as clear without the comments.

Reply:
The comments are there so Sphinx can render the documentation. Otherwise these values would not show up at all.

Reply:
I see, seems a shame it works this way.